pegasus/sites.v3/hourofcode.com/public/thanks.md.partial

```diff
@@ -0,0 +1,50 @@
+---
+title: Thanks for signing up to host an Hour of Code!
+layout: wide
+nav: how_to_nav
+
+social:
+  "og:title": 'Hour of Code: Join the Movement'
+  "og:description": 'The Hour of Code is a global movement reaching tens of millions of students of all ages in 180+ countries and over 45 languages.'
+  "og:image": "http://{{ urls/request_host }}/images/hourofcode-2015-video-thumbnail.png"
```

Comment:
@breville Embedding the partial in the metadata is not working. See screenshot below:

<img width="492" alt="Screen Shot 2019-08-09 at 9 40 11 AM" src="https://user-images.githubusercontent.com/30066710/62794753-bd41c200-ba89-11e9-8751-6df545e8939e.png">

Is it ok to hard code the url ("http://hourofcode.com")?

Reply:
As discussed, we would rather not expose raw URLs to the translation pipeline (and this file is translated) and we also like to be able to test this metadata on non-production environments.  Fortunately we can just add this page to https://github.com/code-dot-org/code-dot-org/blob/b092c48f87325fdc7d0341734eae43a3c251ddbf/pegasus/src/social_metadata.rb.

Reply:
Thanks Brendan!